openshift/wps-api-base/docker/Dockerfile

```diff
@@ -10,6 +10,9 @@ ARG DOCKER_IMAGE=artifacts.developer.gov.bc.ca/github-docker-remote/osgeo/gdal:u
 # docker build --build-arg DOCKER_IMAGE=ubuntu:24.04 . --tag=wps-api-base:ubuntu.24.04-latest
 FROM ${DOCKER_IMAGE}
```

Comment:
What I mean in this [comment](https://github.com/bcgov/wps/pull/4032#issuecomment-2420384472) was just:
```suggestion
FROM ghcr.io/osgeo/gdal:ubuntu-small-3.9.2
```

Reply:
oh derp, I get what you mean now, I thought there was other custom stuff going on - didnt realize it was just using another ghcr repo!

Reply:
I can probably update the workflow too since it won't have to pass in the base image name any longer

Reply:
Do you want the open shift `build.yaml` to be update to reflect this change too?

Reply:
Yeah that'd be great

Reply:
maybe its better to leave the base image to be configurable, but just update the base image on the workflow to be: `ghcr.io/osgeo/gdal:ubuntu-small-3.9.2`

Reply:
ie) this line: https://github.com/bcgov/wps/pull/4032/files#diff-581854468d04e2aa749d259292a93a295bda3c15f20a399d85fa8e80b211f163R30

Reply:
Just because I'm not sure if anything else you guys have is using the `DOCKER_IMAGE` arg or something

Reply:
Either way works for me.